main-settings/src/main/scala/sbt/InputTask.scala

```diff
@@ -27,7 +28,7 @@ final class InputTask[T] private (val parser: State => Parser[Task[T]]) {
         Parser.parse(in, parser(s)) match {
           case Right(v) => Parser.success(v)
           case Left(msg) =>
-            val indented = msg.lines.map("   " + _).mkString("\n")
+            val indented = new StringOps(msg).lines.map("   " + _).mkString("\n")
```

Comment:
i'd put all those repeated `new StringOps(msg).lines` in a utility function where they are easy to find  later

Reply:
https://github.com/scala/bug/issues/11125

Reply:
I guess I'll just wait till 2.12.7 comes out.